client/src/components/breakdown/Breakdown.jsx

```diff
@@ -0,0 +1,16 @@
+
+import React,{ useState,useEffect } from "react";
+import axios from "axios"
+import Category from "../category"
+import "/Breakdown.css"
+
+
+function Breakdown() {
+    const [breakdown, setBreakdown] = useState([]);
+    useEffect(()=>{
+        axios.get('http://localhost:8080/categories/breakdown').then(res=>{setBreakdown(res.data)})
```

Comment:
extract the base url (http://localhost:8080/) to another file and use it in the API calls